pipelines/build-from-scratch.yaml

```diff
@@ -0,0 +1,390 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: build-from-scratch
+  namespace: okd
+spec:
+  params:
+    - name: result-mirror-location
+      type: string
+      description: Location to mirror resulting images
+      default: quay.io/vrutkovs/okd-release
+    - name: result-image
+      type: string
+      description: Resulting release location
+      default: quay.io/vrutkovs/okd-release:4.12.0-0.okd-centos9-build-from-scratch
+    - name: result-image-name
+      type: string
+      description: Internal release name
+      default: 4.12.0-0.okd-centos9-build-from-scratch
+  workspaces:
+    - name: push-credentials
+  tasks:
+    - name: prepare-build-images
+      runAfter:
+      - initial-import
```

Comment:
There is no initial-import task here, should we get rid of this dep?

Reply:
Good point, removed